lib/octodown/support/pdf_file.rb

```diff
@@ -0,0 +1,29 @@
+require 'tempfile'
+require 'fileutils'
+require 'pdfkit'
+
+module Octodown
+  module Support
+    class PDFFile < Tempfile
+      def self.create(content)
+        document = new ['octodown', '.pdf']
+        document.persistent_write content
+      end
+
+      def persist
+        ObjectSpace.undefine_finalizer self
+        self
+      end
+
+      def persistent_write(content)
+        write convert_to_pdf(content)
+        close
+        persist
+      end
+
+      def convert_to_pdf(content)
+        ::PDFKit.new(content).to_pdf
```

Comment:
Since PDFKit requires an external library to run, we should rescue this statement with a message about install wkhtmltopdf.

```ruby
begin
  ::PDFKit.new(content).to_pdf
rescue StandardError
  STDERR.puts 'wkhtmltopdf must be installed in order to use the --pdf option'
  STDERR.puts 'Try: gem install wkhtmltopdf-binary'
  exit 1
end
```